src/sources/kubernetes_logs/mod.rs

```diff
@@ -74,9 +77,12 @@ pub struct Config {
     /// Override global data_dir
     data_dir: Option<PathBuf>,
 
-    /// Specifies the field names for metadata annotation.
+    /// Specifies the field names for Pod metadata annotation.
     annotation_fields: pod_metadata_annotator::FieldsSpec,
```

Comment:
I think it would be worth renaming this field to `pod_annotation_fields` so it is consistent with `namespace_annotation_fields`. You could alias it to `annotation_fields` so that it don't break backward compatibility..